t/op/array.t

```diff
@@ -619,14 +685,19 @@ $#a = -1; $#a++;
     is @a, 0,'unwinding localization of elem past end of array shrinks it';
 
     # Again, but with a package array
+    no warnings 'shadow';
     package tmp; (\our @a)->$#*++; package main;
     my @b = @a;
+    use warnings 'shadow';
     ok !exists $a[0], 'copying an array via = does not vivify elements';
     delete $a[0];
     @a[1..5] = 1..5;
     $#a++;
-    my $count;
-    my @existing_elements = map { exists $a[$count++] ? $_ : () } @a;
```

Comment:
I think that `$count = 0` would have been a simpler solution here
alternatively we could have use another block code

Reply:
Okay.